src/cli/commands/force/lightning/lwc/test/utam/run.ts

```diff
@@ -0,0 +1,235 @@
+        `$ force:lightning:lwc:test:utam:run -p iOS --config '/path/to/wdio.conf.js' --spec '/path/to/myTest.spec.js'`,
+        `$ force:lightning:lwc:test:utam:run -p Android --config '/path/to/wdio.conf.js' --spec '/path/to/myTest.spec.js'`
+    ];
+
+    public static cmd = `npx wdio --version`;
+
+    private static createError(stringId: string): SfError {
+        return new SfError(
+            messages.getMessage(stringId),
+            'lwc-dev-mobile',
+            Run.examples
+        );
+    }
+
+    public static flagsConfig = {
+        ...CommandLineUtils.createFlagConfig(FlagsConfigType.Platform, true),
+        config: flags.string({
+            description: messages.getMessage('configFlagDescription'),
+            required: true,
+            validate: (wdioConfig) => {
+                if (wdioConfig && wdioConfig.trim().length > 0) {
+                    return true;
+                } else {
+                    throw Run.createError('error:invalidConfigFlagDescription');
+                }
+            }
+        }),
+        spec: flags.string({
+            description: messages.getMessage('specFlagDescription'),
+            required: true,
+            validate: (testPath) => {
+                if (testPath && testPath.trim().length > 0) {
+                    return true;
+                } else {
+                    throw Run.createError('error:invalidSpecFlagDescription');
+                }
+            }
+        })
+    };
+
+    private _requirements: CommandRequirements = {};
+    public get commandRequirements(): CommandRequirements {
+        if (Object.keys(this._requirements).length === 0) {
+            const requirements: CommandRequirements = {};
+            requirements.setup = CommandLineUtils.platformFlagIsAndroid(
+                this.flags.platform
+            )
+                ? new AndroidEnvironmentRequirements(this.logger)
+                : new IOSEnvironmentRequirements(this.logger);
+            this._requirements = requirements;
+        }
+
+        return this._requirements;
+    }
+
+    public async init(): Promise<void> {
+        if (this.logger) {
+            // already initialized
+            return Promise.resolve();
+        }
+
+        CommandLineUtils.flagFailureActionMessages = Run.examples;
+        return super
+            .init()
+            .then(() => Logger.child('force:lightning:lwc:test:utam:run', {}))
+            .then((logger) => {
+                this.logger = logger;
+                return Promise.resolve();
+            });
+    }
+
+    public async run(): Promise<any> {
+        this.logger.info(`UTAM run command invoked.`);
+
+        return RequirementProcessor.execute(this.commandRequirements)
+            .then(async () => {
+                this.logger.info(
+                    `Setup requirements met, continuing with running a UTAM test.`
+                );
+                return this.executeWdioAvailabilityCheck();
```

Comment:
`executeWdioAvailabilityCheck` should be turned into a class that extends from `Requirement` and then you should add that to `commandRequirements` along with other requirements. Please refer to [this code](https://github.com/forcedotcom/lwc-dev-mobile/blob/7baa9708334368740f0cb93649baf2a92925bd7e/src/cli/commands/force/lightning/local/device/create.ts#L163) as an example.

Reply:
Per discussion, not checking wdio anymore.